packages/inspection-capture-web/src/PhotoCapture/PhotoCapture.tsx

```diff
@@ -140,6 +144,8 @@ export function PhotoCapture({
   const loading = useLoadingState();
   const addDamageHandle = useAddDamageMode();
   useComplianceAnalytics({ inspectionId, sights });
+  useAppStateMonitoring({ authToken, inspectionId, vehicleType, steeringWheel });
+  useAppStateAnalytics({ inspectionId });
```

Comment:
And I think these two hooks should not be exported and only be used inside the `common` package